docs/usage.md

```diff
@@ -157,6 +157,12 @@ to the pytest.
 * `--install-lvmo` - Deploy LVMCluster, will skip ODF deployment.
 * `--lvmo-disks` - Number of disks to add to SNO deployment.
 * `--lvmo-disks-size` - Size of disks to add to SNO deployment.
+* `--disable-environment-checker` - Disable the leftover checks in existing flow.
+* `--resource-checker` - This will identify the leftover which was created by test cases. This is
+    similar to environment-checker, only difference is resource-checker will track the resources
+    created during test case run whereas environment-checker will track all resources in
+    cluster irrespective of who created.
+* `--kubeconfig` - Location of kubeconfig.
```

Comment:
We should probably overwrite env variable KUBECONFIG pointing to this specific location as well like we do here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/__init__.py#L208

But probably on the place in `ocs_ci/framework/pytest_customization/ocscilib.py`.

Reply:
done